charts/cloudzero-agent/values.yaml

```diff
@@ -47,7 +47,7 @@ prometheusConfig:
 kube-state-metrics:
   enabled: false
   extraArgs:
-    - --metric-labels-allowlist=namespaces=[*],pods=[*],deployments=[app.kubernetes.io/*,k8s.*]
+    - --metric-labels-allowlist=pods=[app.kubernetes.io/version,app.kubernetes.io/name,app.kubernetes.io/instance]
```

Comment:
one potential downside of even this is that these tags can have pretty high cardinality. maybe we should only have `app.kubernetes.io/version`?